controllers/course_memberships_controller.js

```diff
@@ -0,0 +1,58 @@
+const db = require('../db');
+const sql = require('../sql');
+const log = require('../helpers/logging');
+const { coursePath } = require('../routes/helpers/courses');
+const { courseMembershipsPath } = require('../routes/helpers/course_memberships');
+
+exports.index = (req, res, next) => {
+  const { semester_name } = req.params;
+  const { module_code } = req.params;
```

Comment:
Line 8-9 can be simplified to:
```suggestion
  const { semester_name, module_code } = req.params;
```
Same for other similar cases